src/Launcher.vue

```diff
@@ -7,7 +7,7 @@
       :style="{backgroundColor: colors.launcher.bg}"
       @click.prevent="isOpen ? close() : openAndFocus()"
     >
-      <div v-if="newMessagesCount > 0 && !isOpen" class="sc-new-messsages-count">
+      <div v-if="newMessagesCount > 0" class="sc-new-messsages-count">
```

Comment:
` && !isOpen` is removed because I now show unread count always, which is useful when there are many chat rooms.